docs/content/doc/help/faq.en-us.md

```diff
@@ -0,0 +1,246 @@
+
+### Only allow/block certain email domains
+If using OpenID, you can configure `WHITELISTED_URIS` or `BLACKLISTED_URIS` in your `app.ini`  
+**NOTE:** whitelisted takes precedence, so if it is non-blank then blacklisted is ignored
+
+### Issue only users
+The current way to achieve this is to create/modify a user with a max repo creation limit of 0.
+
+### Enable Fail2ban
+
+Use [Fail2Ban]({{ relref "doc/usage/fail2ban-setup.md" >}}) to monitor and stop automated login attempts or other malicious behavior based on log patterns
+
+## How to add/use custom themes
+Gitea supports two official themes right now, `gitea` and `arc-green` (`light` and `dark` respectively)  
+To add your own theme, currently the only way is to provide a complete theme (not just color overrides)  
+  
+As an example, let's say our theme is `arc-blue` (this is a real theme, and can be found [in this issue](https://github.com/go-gitea/gitea/issues/6011))  
+Name the `.css` file `theme-arc-blue.css` and add it to your custom folder in `custom/pulic/css`  
+Allow users to use it by adding `arc-blue` to the list of `THEMES` in your `app.ini`
+
+## SSHD vs built-in SSH
+SSHD is the built-in SSH server on most Unix systems.  
+Gitea also provides its own SSH server, for usage when SSHD is not available.
+
+## Gitea is running slow
+The most common culprit for this is loading federated avatars.  
+This can be turned off by setting `ENABLE_FEDERATED_AVATAR` to `false` in your `app.ini`  
+Another option that may need to be changed is setting `DISABLE_GRAVATAR` to `true` in your `app.ini`
+
+## Can't create repositories/files
+Make sure that Gitea has sufficient permissions to write to its home directory and data directory.  
+See [AppDataPath and RepoRootPath](#where-does-gitea-store-x-file)
+
+**Note for Arch users:** At the time of writing this, there is an issue with the Arch package's systemd file including this line:
+`ReadWritePaths=/etc/gitea/app.ini`  
+Which makes all other paths non-writeable to Gitea.
+
+## Translation is incorrect/how to add more translations
+Our translations are currently crowd-sourced on our [Crowdin project](https://crowdin.com/project/gitea)  
+Whether you want to change a translation or add a new one, it will need to be there as all translations are overwritten in our CI via the Crowdin integration.
+
+## SSH issues
+
+If you cannot reach repositories over `ssh`, but `https` based git repository access works fine, consider looking into the following.
+
+First, make sure you can access Gitea via SSH.  
+`ssh -i ~/.ssh/<keyfile> git@myremote.example`  
+If the connection is successful, you should receive an error message like the following:
+```
+Hi there, You've successfully authenticated, but Gitea does not provide shell access.
+If this is unexpected, please log in with password and setup Gitea under another user.
+```
```

Comment:
@zeripath Is this what you were referring to?

Reply:
Yup. But say that if you don't receive that you're not logging in with a key under Gitea's control and hooks won't run. (Maybe it could be under a Why won't hooks run section?) I could probably add that after this is merged though.

Reply:
I will add a section for hooks and reference this bit.

Reply:
Done. It references this and mentions the `Resynchronize pre-receive, update and post-receive hooks of all repositories.` option.